CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Added Validation when S3 endpoint had a prefix of bucket name. #3414
```

Comment:
I would say this is an ENHANCEMENT, since we're not adding a new restriction, we're just validating it earlier, WDYT? 
Also, please move the changelog entry at the end of the category.

Reply:
@colega I've fixed.